client/homebrew/pages/archivePage/archivePage.jsx

```diff
@@ -0,0 +1,443 @@
+        });
+
+        if (title !== '') {
+            try {
+                await request
+                    .get(
+                        `/api/archive/total?title=${title}&v3=${v3}&legacy=${legacy}`
+                    )
+                    .then((response) => {
+                        if (response.ok) {
+                            this.setState({
+                                totalBrews: response.body.totalBrews,
+                            });
+                        }
+                    });
+            } catch (error) {
+                console.log('error at loadTotal: ', error);
+                this.setState({ error: `${error.response.status}` });
+                this.updateStateWithBrews([], 1);
+            }
+        }
+    },
+
+    renderNavItems: function () {
+        return (
+            <Navbar>
+                <Nav.section>
+                    <Nav.item className="brewTitle">
+                        Archive: Search for brews
+                    </Nav.item>
+                </Nav.section>
+                <Nav.section>
+                    <NewBrew />
+                    <HelpNavItem />
+                    <RecentNavItem />
+                    <Account />
+                </Nav.section>
+            </Navbar>
+        );
+    },
+
+    validateInput: function (e) {
+        const textInput = e.target
+        const submitButton = document.getElementById('searchButton');
+        if (textInput.value.length > 3) {
+            submitButton.disabled = false;
+        } else {
+            submitButton.disabled = true;
+        }
+    },
+
+    renderForm: function () {
+        return (
+            <div className="brewLookup">
+                <h2 className="formTitle">Brew Lookup</h2>
+                <div className="formContents">
+                    <label>
+                        Title of the brew
+                        <input
+                            id="title"
+                            type="text"
+                            name="title"
+                            defaultValue={this.state.title}
+                            onKeyUp={(e) => {
+                                this.validateInput(e);
+                            }}
+                            onKeyDown={(e) => {
+                                if (e.key === 'Enter') {
+                                    this.loadTotal();
+                                    this.loadPage(1, true);
+                                }
+                            }}
+                            placeholder="v3 Reference Document"
+                        />
+                    </label>
+                    <small>
+                        Tip! you can use <code>-</code> to negate words, and{' '}
+                        <code>"word"</code> to specify an exact string.
+                    </small>
+                    <label>
+                        Results per page
+                        <select name="pageSize" id="size">
```

Comment:
This still seems like way too much granularity in options.  Just because it's possible to return many different amounts, it seems needless to offer it.  

And I would say that '6' is a weird number to increment by.  I understand that an *even* number is good, because currently it's returning as predominantly a 2 column list.  But that may not always be the case (for example, if the screen is narrow or we later change how this list is displayed).  And why would anyone only ever want 6 as a max number of results back?  Is someone really getting any benefit from choosing between 6 and 10 results, would a difference greater than 4 break the bank?

If an even number is desired, stick to 10's....but actually 20's.   Options being `20`, `40`, `60`, dropping `60` if it is too much of a toll on the server.  

Sidenote-- I am doing a search for "druid" which returns 222 brews, or 23 pages with 6 brews each.  If i change the `size` parameter in the address bar manually to `60`, it returns only 6 results at a time *but the "pages" portion under the results only shows 4 pages*.  So somewhere the app knows I am only supposed to 4 pages of 60 results each, but returns only 6 brews on each (for 24 brews total).  The param value in the URL should be the source of truth for everything in the query, I think.

Reply:
6 is the only number of results that fit into my screen without needing to scroll, that's where that number comes from. But i'm not too attached to the idea.

Reply:
good catch with the page number, it is still getting them from the input. I'll fix this.

Reply:
The problem with this is i'm supposed to update state if the form is updated, then get from the state, but the setstate method is not fast enough to keep up, and i get the last search instead, not sure how to account for that.